### ISSUE TITLE
Jobs can be in limbo after a worker failure

### ISSUE THREAD

**Issue by BruceBGordon** [yahoo/guerilla#16]
When a worker fails (cancelled for example), a job can be on stuck on the queue. A mildly effective fix would be for a worker at startup to see if any jobs are assigned to it and mark them as cancelled.

**Comment by thekushpatel** [comment]
Agreed. So my idea for this was to have a Cleanup class that would have multiple functions including one that did exactly what you mentioned. Another function would be to remove files from the disk for jobs that no longer exist. These functions would be called on startup and could also be called on some schedule.
